### PR TITLE
Make compatible with gym 0.23.1

### PR DESCRIPTION
**Pull request by nicoguertler**
## Description

Adapted to gym version 0.23.1. The following changes were made:

* Adapt setup.py.
* Replace GoalEnv with Env.
* Explicitly set data type of Box spaces to np.float64 (as observations are np.float64 and gym does not ignore this in `contains` anymore).
* Imported gym_wrappers to be able to make reach-v0 environment with gym.make.

## How I Tested

Ran tests with pytest.

### CODE REVIEW

**Review by luator — CHANGES_REQUESTED** [review]
Thanks a lot for fixing this!  The PR needs a bit of cleanup but apart from this it looks good to me.

- There are several `*.swp` files added, which I assume shouldn't be here.  Maybe it would make sense to add it to .gitignore?
- There are some formatting issues (see failing check), please run `black` to fix them.

File: tests/test_determinism.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import gym
 
+import trifinger_simulation.gym_wrapper
```
[the remark this answers is not in the document]
Comment (nicoguertler) [reply tests/test_determinism.py]: You are right, I removed the superfluous import.

**Comment by nicoguertler** [comment]
Sorry about the swap files. I removed them and added *.swp to .gitignore. I also ran black.

File: .flake8 [hunk 2]
```diff
@@ -1,3 +1,4 @@
 [flake8]
 max-line-length=79
 ignore=E203,W503,E501
+per-file-ignores = tests/test_determinism.py:F401
```
Comment (luator) [inline .flake8:4]: This is probably not needed anymore?

Comment (nicoguertler) [reply .flake8]: Yes, you are right, sorry.

**Review by luator — APPROVED** [review]

**Comment by luator** [comment]
Thanks a lot!

**Pull request merged**